presto-cache/src/main/java/com/facebook/presto/cache/filemerge/FileMergeCacheManager.java

```diff
@@ -140,6 +150,15 @@ public FileMergeCacheManager(
                 }
             }));
         }
+
+        this.cacheSizeCalculateExecutor.scheduleAtFixedRate(
+                () -> {
+                    cacheScopeFiles.keySet().forEach(cacheIdentifier -> cacheScopeSizeInBytes.put(cacheIdentifier, getCacheScopeSizeInBytes(cacheIdentifier)));
+                    cacheScopeSizeInBytes.keySet().removeIf(key -> !cacheScopeFiles.containsKey(key));
+                },
+                0,
+                15,
```

Comment:
It is not. Because this is more of an experimental feature to foresee all the work we would need from alluxio and also finalize on the communication protocol and configs from our side